packages/xdpyinfo.rb

```diff
@@ -0,0 +1,30 @@
+# Adapted from Arch Linux xorg-xdpyinfo PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/xorg-xdpyinfo/trunk/PKGBUILD
+
+require 'package'
+
+class Xorg_xdpyinfo < Package
+  description 'Display information utility for X'
+  version '1.3.2-4'
```

Comment:
Why do we need `-4`?

Reply:
Sorry that was a straight import from the arch pkgbuild, and I hadn't reset the -4.